src/cdi_ipcress/IpcressDataTable.cc

```diff
@@ -107,13 +107,6 @@ void IpcressDataTable::setIpcressDataTypeKey() const {
         ipcressDataTypeKey = "ragray";
         dataDescriptor = "Gray Rosseland Absorption";
         break;
-      // NOTE: THIS KEY DOES NOT ACTUALLY EVER EXIST, See LA-UR-01-5543
```

Comment:
Do we want to keep a reference to this LA-UR somewhere? Something like `// To see what keys exist, see LA-UR-01-5543, "TITLE".`

Reply:
Good idea.  I'll add the full reference to the class header.

Reply:
Fixed in 5a93fd0.